src/repoze/filesafe/__init__.py

```diff
@@ -13,6 +13,7 @@ def _remove_manager(*a):
     except AttributeError:
         pass
 
+
```

Comment:
Please don't run a `pep8` tool on the sources when submitting a non-janitorial PR.